spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/service/DeployerInitializationService.java

```diff
@@ -61,4 +81,12 @@ public void initialize(ApplicationReadyEvent event) {
 			});
 		});
 	}
+
+	private boolean singleDeployerExists() {
+		int deployersCount = 0;
+		for (Platform platform: this.platforms) {
+			deployersCount = deployersCount + platform.getDeployers().size();
+		}
+		return (deployersCount > 1) ? false : true;
```

Comment:
Just to make sure, if `deployersCount` is still zero at this point, do logic around `singleDeployerExists()` still work?